eks-worker.tf

```diff
@@ -5,13 +5,13 @@
 module "aws_eks_managed_node_groups" {
   source = "./modules/aws-eks-managed-node-groups"
 
-  for_each = { for key, value in var.managed_node_groups : key => value
-    if length(var.managed_node_groups) > 0
-  }
+  for_each = var.managed_node_groups
 
   managed_ng = each.value
   context    = local.node_group_context
 
+  cluster_version = var.cluster_version
```

Comment:
We already have a `cluster_version` variable in `local.node_group_context`. We just need to leverage that
https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/093574f79a50413b961376d9ec0310bda6b514b9/modules/aws-eks-managed-node-groups/variables.tf#L13